### DISCUSSION TITLE
Invalid verification code throws an exception with a 200 status code

### DISCUSSION THREAD

**Discussion by lukeyouell** [craftcms/cms#8789] [Ideas]
### Description

When an invalid verification code is used by a user, an exception is thrown with a status code of 200 which contradicts the fact that an exception has been thrown. 

https://github.com/craftcms/cms/blob/develop/src/controllers/UsersController.php#L1920

This also raises the question of whether or not this is worthy of an exception in the first place? I can see arguments for and against doing so, but in our case this error is reported to Sentry - which we don't really need to be notified of.

**Comment by brandonkelly** [comment]
Agree this should probably be cleaned up.

**Comment by lukeyouell** [comment]
Thanks Brandon!

I've since spotted a few other exceptions being thrown with 200 status codes, linked to below for your reference:

https://github.com/craftcms/cms/blob/2bd24e66d74742ead9a5c18f28e1fa5f0ad5bea3/src/web/Application.php#L204

https://github.com/craftcms/cms/blob/2bd24e66d74742ead9a5c18f28e1fa5f0ad5bea3/src/web/Application.php#L621

https://github.com/craftcms/cms/blob/a38c3bee4f62b3a3e0bc9083d8c475624358d349/src/services/Plugins.php#L243
